recipe/meta.yaml

```diff
@@ -1,21 +1,31 @@
 package:
-  name: msinttypes
-  version: r26
+    name: msinttypes
+    version: r26
 
 source:
-  url: https://msinttypes.googlecode.com/files/msinttypes-r26.zip 
-  fn: msinttypes-r26.zip
+    url: https://msinttypes.googlecode.com/files/msinttypes-r26.zip
+    fn: msinttypes-r26.zip
+    md5: 94c6acec78be290fc72cd01f2755341a
 
-build:  # [not win]
-  skip: True  # [not win]
+build:
+    number: 1
+    skip: True  # [not win]
+    features:
```

Comment:
So ugly 😱 
I am eager to remove these ASAP... I know there are things in the pipeline, I can only wish they get resolved 😄